tests/test_custom_router.py

```diff
@@ -81,3 +87,31 @@ def add_route(self, uri_template, method_map, resource, name):
 
         self.assertEqual(len(check), 2)
         self.assertIn('my-url-name-arg', check)
+
+    def test_custom_router_takes_req_positional_argument(self):
+        def resource(req, resp):
```

Comment:
Nitpick: This is a specific HTTP method responder for a resource, not the resource itself, so perhaps this (and the function on L106) should be renamed to `resource_responder`, `responder`, or similar?